ls1intum/Artemis
# Make the remaining management buttons work like links

### Checklist


### Motivation and Context

Continues #2669. All other "buttons" are not router links and would need even further refactoring.

### Steps for Testing

The following buttons did change and work like links now:
- Learning Goals: "Edit", "Create"
- (Test) Exam summary: "View Results"
- Exam management: "Exam checklist"
- Text & File Upload Instructor Assessment (Submissions) Dashboard (Go to "Instructor Dashboard" then to "Submissions"): "Open Assessment"
- Programming exercises: "Edit in Editor", "Check Plagiarism", "Generate a new Programming Exercise"
- Exercise Hints: "Edit"
- Exercise Scores: "Participations", "View Assessment"
